x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/utils/utils.test.ts

```diff
@@ -451,11 +461,14 @@ describe('utils', () => {
         to: 'now',
         maxSignals: 20,
         ruleExecutionLogger,
+        getTimeRange: getTimeRangeExecutorService,
       });
       const someTuple = tuples[0];
       expect(moment(someTuple.to).diff(moment(someTuple.from), 's')).toEqual(30);
       expect(tuples.length).toEqual(1);
       expect(remainingGap.asMilliseconds()).toEqual(0);
+      expect(someTuple.to.utc().format()).toEqual('2020-01-01T06:05:36Z');
```

Comment:
I added these `expect` statements before updating the function to use the `getTimeRange` callback to ensure that the timestamps returned all stayed the same